api/apps/geoprocessing/src/modules/planning-area/marxan-geo-json.ts

```diff
@@ -0,0 +1,32 @@
+import { BBox, GeoJSON } from 'geojson';
+import { PuSizes } from './pu-sizes';
+
+export class MarxanGeoJson {
```

Comment:
`marxan` isn't giving any idea what it is. What consumers use it for?

Reply:
There is also another "Marxan" GeoJson which holds cost & puId of each geometry inside.

Reply:
where?

this one is specific to the planning area, adding something custom to GeoJSON and serializing it in toJSON,

just encapsulates a bit where GeoJSON keeps bbox and pu sizes

it's used in PlanningAreaDto, and can be used in other dtos when needed

Reply:
`api/apps/geoprocessing/src/modules/surface-cost/adapters/pu-cost-extractor.ts`

Reply:
so it has a different purpose